bats/creds.bats

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bats
+load $BATS_TEST_DIRNAME/helper/common.bash
+
+setup() {
+    setup_common
+}
+
+teardown() {
+    teardown_common
+}
+
+@test "ls empty creds" {
+    run dolt creds ls
+    [ "$status" -eq 0 ]
+    [ "${#lines[@]}" -eq 0 ]
+}
+
+@test "ls new cred" {
+    run dolt creds new
```

Comment:
If you just check $status -eq 0 you don't need run. Just remove.

Reply:
Thanks. Fixed!